main/acle.md

```diff
@@ -1550,70 +1550,70 @@ be found in [\[BA\]](#BA).
-| `__ARM_FEATURE_COPROC` [[ssec-CoProc]](#ssec-CoProc)                                           | Coprocessor Intrinsics                                                                             | 1           |
-| `__ARM_PCS` [[ssec-Pcs]](#ssec-Pcs)                                                            | Arm procedure call standard (32-bit-only)                                                          | 0x01        |
-| `__ARM_PCS_AAPCS64` [[ssec-Pcs]](#ssec-Pcs)                                                    | Arm PCS for AArch64.                                                                               | 1           |
-| `__ARM_PCS_VFP` [[ssec-Pcs]](#ssec-Pcs)                                                        | Arm PCS hardware FP variant in use (32-bit-only)                                                   | 1           |
-| `__ARM_FEATURE_RNG` [[ssec-rng]](#ssec-rng)                                                    | Random Number Generation Extension (Armv8.5-A)                                                     | 1           |
-| `__ARM_ROPI` [[ssec-Pic]](#ssec-Pic)                                                           | Read-only PIC in use                                                                               | 1           |
-| `__ARM_RWPI` [[ssec-Pic]](#ssec-Pic)                                                           | Read-write PIC in use                                                                              | 1           |
-| `__ARM_SIZEOF_MINIMAL_ENUM` [[ssec-Imptype]](#ssec-Imptype)                                    | Size of minimal enumeration type: 1 or 4                                                           | 1           |
-| `__ARM_SIZEOF_WCHAR_T` [[ssec-Imptype]](#ssec-Imptype)                                         | Size of `wchar_t`: 2 or 4                                                                          | 2           |
-| `__ARM_WMMX` [[ssec-WMMX]](#ssec-WMMX)                                                         | Wireless MMX extension (32-bit-only)                                                               | 1           |
+| `__ARM_32BIT_STATE` [[ssec-ATisa]](#a32t32-instruction-set-architecture)                                                | Code is for AArch32 state                                                                          | 1           |
+| `__ARM_64BIT_STATE` [[ssec-ATisa]](#a32t32-instruction-set-architecture)                                                | Code is for AArch64 state                                                                          | 1           |
+| `__ARM_ACLE` [[ssec-TfACLE]](#testing-for-arm-c-language-extensions)                                                     | Indicates ACLE implemented                                                                         | 101         |
```

Comment:
Does this work on the PDF? Don't you have also to realign all the borders of the table?

Reply:
You don't actually. https://www.markdownguide.org/extended-syntax/
It's just good for readability I think.